src/pages/index.jsx

```diff
@@ -20,8 +20,17 @@ export default function Home() {
       <HomepageHeader />
 
       <main>
+        <section className="section flex">
+          <Link to="/docs/latest/Guides/Getting-Started" className="button button--lg button--primary">
+            Getting Started
+          </Link>
+          <Link to="/resources" className="button button--lg button--secondary">
+            Learn More
```

Comment:
could we make these 2 buttons with the same width?

Reply:
Of course, I'll give it a fixed width.